src/net_processing.cpp

```diff
@@ -2745,7 +2745,7 @@ bool ProcessMessage(CNode& pfrom, const std::string& msg_type, CDataStream& vRec
         }
 
         LOCK(cs_main);
-        if (::ChainstateActive().IsInitialBlockDownload() && !pfrom.HasPermission(PF_NOBAN)) {
+        if (::ChainstateActive().IsInitialBlockDownload() && !pfrom.HasPermission(PF_DOWNLOAD)) {
```

Comment:
This exception was strange to begin with. But I don't think it has anything to do with `download` permission. Maybe keep it `noban`?

Reply:
Why is is strange? The literal motivation for this is that nodes with this permission can use us "as a block source" (#6974).

Imagine you spin up an archival node, disconnect it from the network, but make it available in you internal network. Nodes with the download permission should be able to use your archival node as a block source, no?

Reply:
Ah, that PR explains the problem. Even nodes that almost synced can be `IsInitialBlockDownload()`, presumably also nodes that have disconnected for a few days? In that case the new permission makes sense. Would be good to explain in a comment.

Reply:
Suggestions would be useful. Keep in mind that the documentation already says:

```
download (allow getheaders during IBD, no disconnect after maxuploadtarget limit)

Reply:
```
// Allow getheaders while IsInitialBlockDownload is true, such as after prolonged disconnection.
```

Documenting that here instead of just in the `bitcoind` help seems useful, especially `IsInitialBlockDownload` has a confusing name.

Reply:
> IsInitialBlockDownload has a confusing name

I think `IsInitialBlockDownload` is self-explanatory, so I will leave the documentation as is for now. I am happy to review follow-up documentation improvements, though.